### PR TITLE
Add more USB tests

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

File: firmware/ja-rule.X/nbproject/configurations.xml [hunk 1]
```diff
@@ -470,7 +602,7 @@
         <targetDevice>PIC32MX675F512H</targetDevice>
         <targetHeader></targetHeader>
         <targetPluginBoard></targetPluginBoard>
-        <platformTool>PICkit3PlatformTool</platformTool>
+        <platformTool>Simulator</platformTool>
```
Comment (peternewman) [inline firmware/ja-rule.X/nbproject/configurations.xml]: Revert this perhaps? I'm not sure how we'll deal with different people with different programmers anyway.

File: firmware/ja-rule.X/nbproject/configurations.xml [hunk 2]
```diff
@@ -740,83 +1028,6 @@
         <property key="wpo-lto" value="false"/>
         <appendMe value="-L&quot;../../linker&quot;"/>
       </C32Global>
-      <PICkit3PlatformTool>
```
Comment (peternewman) [inline firmware/ja-rule.X/nbproject/configurations.xml]: Keep this maybe?

**Comment by peternewman** [comment]
LGTM

**Comment by nomis52** [comment]
reverted the xml file.

**Comment by peternewman** [comment]
LGTM

**Pull request merged**